src/summariser.py

```diff
@@ -2,17 +2,48 @@
 from cohere.types.non_streamed_chat_response import NonStreamedChatResponse
```

Comment:
Why import `cohere` and separately have `from cohere` etc? Just curious if this is intentional or organic

Reply:
This was for type hinting - I'd have to have that big old line just to type hint later on. Looked neater to have just this NonStrramedResponse as an object